src/main/java/controllers/ConsoleInputController.java

```diff
@@ -99,48 +102,66 @@ private void checkRecurring() {
      * @param input : Input typed by user into CLI
      */
     @Override
-    public void onCommandReceived(String input) {
+    public void onCommandReceived(String input) throws DukeException {
         checkRecurring();
+        Scanner scanner = new Scanner(input);
```

Comment:
If you want to create a new Scanner here, remove the `Scanner sc = new Scanner(System.in);` in View